homeassistant/components/transmission/switch.py

```diff
@@ -24,17 +26,19 @@ async def async_setup_entry(
 ) -> None:
     """Set up the Transmission switch."""
 
-    tm_client: TransmissionClient = hass.data[DOMAIN][config_entry.entry_id]
+    coordinator: TransmissionDataUpdateCoordinator = hass.data[DOMAIN][
+        config_entry.entry_id
+    ]
     name: str = config_entry.data[CONF_NAME]
 
     dev = []
     for switch_type, switch_name in SWITCH_TYPES.items():
-        dev.append(TransmissionSwitch(switch_type, switch_name, tm_client, name))
+        dev.append(TransmissionSwitch(switch_type, switch_name, coordinator, name))
 
     async_add_entities(dev, True)
```

Comment:
We should not call `update` during entity addition. The coordinator has already made the first refresh before we set up all the platforms. Remove the second parameter `True` here.